### PR TITLE
Test the CAB locator more thoroughly

### PR DESCRIPTION
**Pull request by stevenwilkin**
This steps through:
1. browsing to '/locations'
2. submitting an invalid postcode
3. submitting a valid postcode

### CODE REVIEW

File: bin/cab_locator_test.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+form['postcode'] = 'SW1A 2HQ'
+page = form.submit(form.buttons.first)
+
+if page.body =~ /Face-to-face locations near/
+  puts '> Renders locations'
+else
+  raise 'Should render locations'
+end
+
+puts '>> OK'
```
Comment (andrewgarner) [inline bin/cab_locator_test.rb]: Should we check the cache headers too? We've lost that check having removed those paths from the previous tests.

Comment (stevenwilkin) [inline bin/cab_locator_test.rb]: This test script is concerned with how the feature works when driven by a browser. Headers don't come into consideration.

`/locations` is still included in the page/header test script which I think gives us enough confidence.

Comment (andrewgarner) [inline bin/cab_locator_test.rb]: :+1:

**Comment by stevenwilkin** [comment]
Updated to click through to location page

**Comment by andrewgarner** [comment]
:+1:

**Pull request merged**